Studio-Lovelies/GG-JointJustice-Unity
# feat(Investigation): Handles #initial talk choices

## Summary
Writers can tag talk choices with `#initial`, which forces the game to play them once, before being offered the choice menu.

Also updates the JetBrains Rider package, as the newer one prevents the debugger from disconnecting and making me restart both Unity and IDE. 🫠 

## Before/after screenshots and/or animated gif

https://github.com/user-attachments/assets/5ca4a66d-8dab-41af-ae4c-1c7db87d11d1

## Testing instructions
1. Open the `Game` scene
2. Set the `GameStarter` to `InvestigationUI`
3. Select `Talk`
4. Notice no menus opening and Dan starting to talk
5. Play through the dialogue and notice the main menu reappearing*
6. Select `Talk`
7. Notice the choice menu opening

*in the final design, the player is brought back to the `Talk` sub-menu instead of the investigation main menu, but as the sub menus currently can't be exited, the player is moved to the main menu to prevent further soft-locks. After a way to exit the menus exists, I will update this behavior.

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - #482
